src/GraphQL.NewtonsoftJson/ExecutionResultContractResolver.cs

```diff
@@ -10,7 +10,7 @@ public class ExecutionResultContractResolver : DefaultContractResolver
         private readonly CamelCaseNamingStrategy _camelCase = new CamelCaseNamingStrategy();
 
         protected override JsonConverter ResolveContractConverter(Type objectType) =>
-            objectType == typeof(ExecutionResult)
+            objectType == typeof(ExecutionResult) || objectType.IsSubclassOf(typeof(ExecutionResult))
```

Comment:
I think this is easier to understand:
```suggestion
            typeof(ExecutionResult).IsAssignableFrom(objectType)
```

Or maybe even
```suggestion
            objectType.IsAssignableTo(typeof(ExecutionResult))
```
with
```c#
internal static class TypeExtensions
{
	internal static bool IsAssignableTo(this Type A, Type b) => b.IsAssignableFrom(a);
}
```

Reply:
OK, `typeof(ExecutionResult).IsAssignableFrom(objectType)`